test/Test/Pos/UtilSpec.hs

```diff
@@ -112,3 +153,38 @@ verifyDiffMapIsSmaller (SmallHashMap hm1) (SmallHashMap hm2) =
         sumValSizes = sum . fmap length
     -- (p || q) <=> ((not p) => q)
     in (null commonKey) || (sumValSizes hm1 > sumValSizes diffMap)
+
+toSingletonN
+    :: (Arbitrary a, Show a, Eq a)
+    => a
+    -> Expectation
+toSingletonN x = one x `shouldBe` (NewestFirst @[] . one $ x)
+
+toSingletonO
+    :: (Arbitrary a, Show a, Eq a)
+    => a
+    -> Expectation
+toSingletonO x = one x `shouldBe` (OldestFirst @[] . one $ x)
+
+
+toFromList
+    :: forall f t a. (Arbitrary (t f a),
+                  Show (t f a),
+                  Eq (t f a),
+                  IL.IsList (f a),
+                  IL.IsList (t f a))
```

Comment:
@Martoon-00 Was this `Each` introduced with the recent changes to `universum`? My compiler's complaining it doesn't exist.